src/components/SearchInput.vue

```diff
@@ -67,6 +67,9 @@ export default {
     axios('/search.json').then(response => {
       this.posts = response.data
     })
+    .catch(function (error) {
```

Comment:
Can you make it an arrow function to keep it consistent?

Reply:
Done.